tests/execute/basic/test.sh

```diff
@@ -2,6 +2,17 @@
 . /usr/share/beakerlib/beakerlib.sh || exit 1
 
 rlJournalStart
+    function check_duration() {
+        local results=$1
+        DURATION_REGEXP="^(((([0-1][0-9])|(2[0-3])):?[0-5][0-9]:?[0-5][0-9]+$))"
+        DURATION=$(grep -A5 "good:" "$results" | grep "duration:" | awk '{print $2}')
```

Comment:
Could you assert currect format for all test results? It should be present every time, right?

Reply:
test updated